projects/fusion-native/src/app/app-config.ts

```diff
@@ -165,6 +166,8 @@ export const components = [
     {name: `${PREFIX}-status-label`, componentInstance: StatusLabelComponent},
     {name: `${PREFIX}-toast`, componentInstance: ToastComponent},
     {name: `${PREFIX}-textarea`, componentInstance: TextareaComponent, avoidJsonParse: true},
+    {name: `${PREFIX}-tabs-v2`, componentInstance: TabsComponentV2},
+    {name: `${PREFIX}-dropdown-dual-multi-select`, componentInstance: DropdownDualMultiSelectComponent},
     {name: `${PREFIX}-tabs`, componentInstance: TabsComponent},
-    {name: `${PREFIX}-dropdown-dual-multi-select`, componentInstance: DropdownDualMultiSelectComponent}
```

Comment:
why you removed `dropdown-dual-multi-select`

Reply:
it was not removed, it exists. Don't know why it was marked as removed.